docs/guides/10-deploy/12-upgrade/10-compatibility.md

```diff
@@ -107,6 +107,19 @@ The following is an illustration of current query-meta compatibility:
 
 <img src="/img/deploy/compatibility.excalidraw.png"/>
 
+## Compatibility between databend-query
+
+| Query version      | Backward compatible with |
+|:-------------------|:-------------------------|
+| [1.2.287, 1.2.311) | [1.2.287, 1.2.311)       |
```

Comment:
Why is `1.2.287` so important? And what about the versions before `1.2.287`?

Reply:
I see this:

```
#### Compatibility status
The following is an illustration of current query-meta compatibility:
| `Meta\Query`      | [0.7.59, 0.8.80) | [0.8.80, 0.9.41) | [0.9.41, 1.1.34) | [1.1.34, 1.2.287) | [1.2.287, +∞) |
```

Reply:
This is query-meta compatibility and has nothing to do with this PR.

from `fuse-compat` test it looks like query is compatible with an very old version, right?

Reply:
> Since which, both protobuf and json can be read;
> Since which, protobuf is used to replace json(which is 1.2.311, as it is already described)

Get, in this doc, only need these two since.

Reply:
> This is query-meta compatibility and has nothing to do with this PR.
> 
> 
> 
> from `fuse-compat` test it looks like query is compatible with an very old version, right?

Right. In fuse-compat test, the old version serialize and deserialize role as json.